src/components/Popover/popoverPropTypes.js

```diff
@@ -22,6 +25,7 @@ const defaultProps = {
 
     animationIn: 'fadeIn',
     animationOut: 'fadeOut',
+    animationInTiming: 300,
```

Comment:
@Uros787 Same replace this with `CONST.ANIMATED_TRANSITION`?

Reply:
yes, done